dashboard/utils.bal

```diff
@@ -126,6 +127,18 @@ function getLoadTestsBadge(string modName) returns string|error {
     return string `[![Load Tests](${badgeUrl})](${repoUrl})`;
 }
 
+function getBalTestNativeBadge(string moduleName) returns string|error {
+    string badgeUrl = string `${GITHUB_BADGE_URL}/workflow/status/${BALLERINA_ORG_NAME}/${moduleName}/Build%20with%20bal%20test%20native?label=`;
+    string repoUrl = string `${BALLERINA_ORG_URL}/${moduleName}/actions/workflows/build-with-bal-test-native.yml`;
+    string workflowFileUrl = string `/${BALLERINA_ORG_NAME}/${moduleName}/master/.github/workflows/build-with-bal-test-native.yml`;
+    http:Response openUrlResult = check openUrl(GITHUB_RAW_LINK, workflowFileUrl).ensureType();
+    string urlResult = check openUrlResult.getTextPayload();
+    if urlResult == "404: Not Found" {
+        badgeUrl = NABADGE;
+    }
+    return string `[![Build with bal test native](${badgeUrl})](${repoUrl})`;
```

Comment:
This is too long for the badge text. Shall we use a small text like this?

```suggestion
    return string `[![Native Test](${badgeUrl})](${repoUrl})`;
```

Reply:
Yes, +1 to reduce the badge text size.

But since most of the standard libs have `native` code (which is Java), IMO the text `Native Test` can be misleading. If this is only for internal tracking purpose +1 for the suggested name.

Reply:
How about Graalvm Check?

Reply:
+1 for `GraalVM Check`
Changed the name via [074823b](https://github.com/ballerina-platform/ballerina-standard-library/pull/3615/commits/074823b15eb79dec30489fc33f2252914c171717)